app/src/scenes/Items/interfaces.ts

```diff
@@ -0,0 +1,35 @@
+export interface ItemStateProps {
+  items: Item[];
+  isLoading: boolean;
+  pagination: {
+    current: number,
+    pageSize: number,
+    total: number,
+  };
+}
+
+export interface Item {
+  id: number;
+  link: string;
+  translations: Language[];
+}
+
+export interface SubCategory {
+  id: number;
+  translations: Language[];
+}
+
+export interface Language {
+  name: string;
+  language: string;
+  description?: string;
```

Comment:
The description is not an optional attribute. isn't it?

Reply:
Create a new interface for item translation (you can extend it from subcategory translation. but let's do not do that because the attribute matches but it isn't a relationship)

Reply:
ok